sdk/src/wallet/operations/block.rs

```diff
@@ -29,17 +31,18 @@ where
             Some(id) => id,
             None => {
                 let current_slot = self.client().get_slot_index().await?;
+                let network_id = self.client().get_network_id().await?;
```

Comment:
this is part of protocol params, so can we just move the call below up and reuse it?

Reply:
Sure